tests/pytorch/test_pytorch_autolog.py

```diff
@@ -2,13 +2,19 @@
 import pytest
 import pytorch_lightning as pl
 import torch
-from iris import IrisClassification, IrisClassificationWithoutValidation
+from examples.pytorch.AxHyperOptimizationPTL.iris import (
+    IrisClassification,
+    IrisClassificationWithoutValidation,
+)
```

Comment:
I think we should avoid importing classes from the examples directory.

Reply:
AX Example needs iris classifiction class. Instead of duplicating the entire class, i moved the iris class inside AX and fixed the tests. 

I can revert it. But the same code (IrisClassification) will be present in both test and also examples. Is that fine ?

Reply:
I think that's fine :)

Reply:
Okie sure . I will  do the same